test/Demangle/Inputs/manglings.txt

```diff
@@ -359,3 +359,4 @@ $sxq_Idgnr_D ---> @differentiable @callee_guaranteed (@in_guaranteed A) -> (@out
 $sxq_Ilgnr_D ---> @differentiable(linear) @callee_guaranteed (@in_guaranteed A) -> (@out B)
 $sS3fIedgyywd_D ---> @escaping @differentiable @callee_guaranteed (@unowned Swift.Float, @unowned @noDerivative Swift.Float) -> (@unowned Swift.Float)
 $syQo ---> $syQo
+$sx1td_t --> $sx1td_t
```

Comment:
@slavapestov -- Do you think this is sufficient to verify that the demangler doesn't crash on a variadic tuple?

Reply:
Yes, it should. But you need to use 3 dashes: --->

Reply:
Thanks for pointing that out.